docs/backend_documentation/profile.md

```diff
@@ -0,0 +1,27 @@
+# Profile
+
+## Current code:  
+@main_blueprint.route("/profile/<string:rcs_id>")  
```

Comment:
when including code blocks you can use https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks. You can also add "python" next to the "```" to highlight python syntax.